echemdb/data/cv/database.py

```diff
@@ -218,5 +205,16 @@ def __getitem__(self, identifier):
             >>> database['alves_2011_electrochemistry_6010_p2_f2a_solid']
             Entry('alves_2011_electrochemistry_6010_p2_f2a_solid')
 
+            >>> database['unvalid_key']
+            Traceback (most recent call last):
+            ...
+            KeyError: "No database entry with key 'unvalid_key'."
+
         """
-        return [entry for entry in self if entry.identifier == identifier][0]
+        entries = [entry for entry in self if entry.identifier == identifier]
+
+        if len(entries) == 0:
+            raise KeyError(f"No database entry with key '{identifier}'.")
+        if len(entries) > 1:
+            raise KeyError(f"The database has more than one entry with key '{identifier}'")
```

Comment:
Can this really happen?

Reply:
I am not sure if we check for duplicates when the database is created. In principle this can happen since the identifier is the filename. Assume the following example:  
`./experiment1/measurement1.yaml`  
`./experiment2/measurement1.yaml`

Reply:
```suggestion
        if len(entries) > 1:
            raise KeyError(f"The database has more than one entry with key '{identifier}'.")
```